.github/workflows/release.yaml

```diff
@@ -50,6 +50,7 @@ jobs:
               echo "Setting Tag Images"
               cd /work
               apt-get update && apt-get install git -y || exit 1
+              git config --global --add safe.directory /work
```

Comment:
can you please make this same pr to;

A)-> price oracle
B) -> apy
C)> el col
D) -> autonomous-fund